Makefile

```diff
@@ -6,11 +6,14 @@ release_drivers=$(foreach d, $(drivers), release/$(d))
 
 all: prepare $(drivers)
 
-prepare: sgp-common/sgp_git_version.c
+prepare: sgp-common/sgp_git_version.c embedded-sht/sht-common/sht_git_version.c
```

Comment:
I would rather go with a phony `prepare-embedded-sht` target, or do we need to list that file explicitly?

Reply:
Probably not. Let me try.

Reply:
Fixed.